src/Confluent.Kafka/AdminClient.cs

```diff
@@ -768,7 +768,8 @@ private SafeKafkaHandle kafkaHandle
         ///     or AdminClient handle.
         /// </param>
         internal AdminClient(Handle handle)
-        {                            
+        {
+            this.cancellationDelayMaxMs = 100;                          
```

Comment:
I believe it should use `ExtractCancellationDelayMaxMs()` so that this value can be configured (and that we reuse the default value).

Reply:
`ExtractCancellationDelayMaxMs` needs Config to be passed on which isn't available with DependentAdminClient as DependentAdminClient is created through librdkafka handle corresponding to the producer/consumer.

Reply:
Or do you mean I should pass an empty config here?

Reply:
Updated to use `ExtractCancellationDelayMaxMs` with an empty config.